packages/store/src/public_api.ts

```diff
@@ -4,7 +4,19 @@ export { Store } from './store';
 export { State } from './decorators/state';
 export { Select } from './decorators/select';
 export { Actions } from './actions-stream';
-export { ofAction, ofActionDispatched, ofActionSuccessful, ofActionCanceled, ofActionErrored } from './operators/of-action';
+export {
+  getSelectorMetadata,
+  getStoreMetadata,
+  ensureStoreMetadata,
+  ensureSelectorMetadata
+} from './internal/internals';
```

Comment:
This metadata stuff never should have been added to the public API!
I really want to remove this again. The only thing on github that is using it is the emitter plugin.
cc @splincode @eranshmil @deebloo